### PR TITLE
Consolidate flow json spec

### PR DESCRIPTION
**Pull request by jotak**
## Description

Consolidate json spec following https://github.com/netobserv/network-observability-console-plugin/pull/383

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [x] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

**Review by skrthomas — APPROVED** [review]
/lgtm

File: docs/flows-format.adoc
Comment (skrthomas) [inline docs/flows-format.adoc:1]: for line 3 & 4: I have this in my file and I would like to keep it that way:
[id="network-observability-flows-format_{context}"]
= Network Flows format reference

For line 10, I have this in my file, and I think it was peer reviewed to be like this, maybe we didn't synch it:
If you are reading this specification as a reference for the Kafka export feature, you must treat all _Labels_ and _Fields_ as regular fields and ignore any distinctions between them that are specific to Loki.

Comment (jotak) [reply docs/flows-format.adoc:1]: sure, this is in the [header](https://github.com/netobserv/network-observability-operator/blob/main/hack/flows-format-header.adoc) file, I'll update it

Comment (jotak) [reply docs/flows-format.adoc:1]: done: https://github.com/netobserv/network-observability-operator/pull/417/commits/8760c99c5331d816e40db9734b4fe3f8cc4d3b2d

**Comment by jotak** [comment]
@skrthomas I fixed the InterfaceDirection xref issue that makes your PR fails https://app.travis-ci.com/github/openshift/openshift-docs/jobs/609884831

**Pull request merged**